week-2/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,19 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) {
+    return "A";
+  }
+  if (mark < 80 && mark > 60) {
+    return "B";
+  }
+  if (mark <= 60 && mark >= 50) {
+    return "C";
+  } else {
+    return "F";
+  }
```

Comment:
You probably don't need this last `else`, just the default `return` statement. Also, do you need to check, `mark < 80`, for example, when the previous statement already deals with any values which are ` >= 80` ?